feature-libs/asm/styles/components/_customer-list.component.scss

```diff
@@ -239,6 +242,9 @@ $sapFioriContent_LabelColor: #6a6d70;
         border-radius: 50%;
       }
     }
+    .cx-cart {
```

Comment:
does this style works for every version?

Reply:
Done